src/main/java/backend/model/payment/SponsoringInvoice.kt

```diff
@@ -99,33 +103,36 @@ class SponsoringInvoice : Invoice {
     }
 
     override fun generatePurposeOfTransfer(): String {
-        TODO()
+        this.purposeOfTransferCode = generateRandomPurposeOfTransferCode()
+        this.purposeOfTransfer = "Spende $purposeOfTransferCode-BREAKOUT-${sponsor.lastname}"
+
+        return this.purposeOfTransfer!!
     }
 
     fun toEmailOverview(): String {
         return """
-        |<b>Challenges</b><br/>
+        |<b>Challenges</b>
         |${this.challenges.toEmailListing()}
-        |<br/><b>Sponsorings</b><br/>
-        |${this.sponsorings.toEmailListing()}<br/>
+        |<b>Sponsorings</b>
+        |${this.sponsorings.toEmailListing()}
         |
-        |<b>Total:</b> $amount<br/>
-        |<b>Already paid:</b> ${amountOfCurrentPayments()}<br/>
+        |<b>Total:</b> $amount
+        |<b>Already paid:</b> ${amountOfCurrentPayments()}
         """.trimMargin("|")
     }
 
     @JvmName("sponsoringToEmailListing")
     private fun List<Sponsoring>.toEmailListing(): String {
-        return this.map { it.toEmailListing() }.foldRight("") { acc, s -> "$acc<br/>\n$s" }
+        return this.map { it.toEmailListing() }.foldRight("") { acc, s -> "$acc\n$s" }
     }
 
     private fun Sponsoring.toEmailListing(): String {
-        return "<b>Team-ID</b> ${this.team?.id} <b>Teamname</b> ${this.team?.name} <b>Status</b> ${this.status} <b>Amount Per Km</b> ${this.amountPerKm} <b>Actual Km</b> ${this.team?.getCurrentDistance()} <b>Billed Amount</b> ${this.billableAmount()}"
+        return "<b>Team-ID</b> ${this.team?.id} <b>Teamname</b> ${this.team?.name} <b>Status</b> ${this.status} <b>Amount Per Km</b> ${this.amountPerKm} <b>Limit</b> ${this.limit} <b>Actual Km</b> ${this.team?.getCurrentDistance()} <b>Billed Amount</b> ${this.billableAmount()}"
```

Comment:
It is printed the way the enum values are labelled. I think this may be helpful for communication between sponsors and support